catalog/transformer/catalog-transformer-streaming-lib/src/test/java/org/codice/ddf/transformer/xml/streaming/lib/TestGenericXmlLib.java

```diff
@@ -119,6 +120,26 @@ public void testNullInputTransform() throws FileNotFoundException, CatalogTransf
     }
 
     @Test
+    public void testNoConfigAndNoFactoriesTransform()
+            throws IOException, CatalogTransformerException {
+        SaxEventHandlerFactory saxEventHandlerFactory = mock(SaxEventHandlerFactory.class);
+        when(saxEventHandlerFactory.getId()).thenReturn("test");
+        SaxEventHandler handler = getNewHandler();
+        when(saxEventHandlerFactory.getNewSaxEventHandler()).thenReturn(handler);
+        XmlInputTransformer xmlInputTransformer = new XmlInputTransformer();
+        xmlInputTransformer.setSaxEventHandlerFactories(Collections.singletonList(
+                saxEventHandlerFactory));
+        MetacardType metacardType = xmlInputTransformer.getMetacardType();
+        assertThat(metacardType.getAttributeDescriptors(),
```

Comment:
Is this assertion part of the same test? If so, we should move it at the end of the method with the other assert. If not, we should consider having two separate tests.